packages/suite-desktop/src-electron/http-receiver.ts

```diff
@@ -121,11 +125,29 @@ export class HttpReceiver extends EventEmitter {
 
         const route = this.routes.find(r => r.pathname === pathname);
         if (route) {
+            response.setHeader('Content-Type', 'text/html; charset=UTF-8');
             // original type has url as optional, Request alias makes it required.
             return route.handler(request as Request, response);
         }
     };
 
+    // TODO: add option to auto-close after X seconds. Possible?
```

Comment:
would be great. But I did not find a way how to do it. Maybe we could add at least a button with xdg-redirect back to Suite

Reply:
I think it was your comment originally that I deleted because I tried to implement the auto-close, just to then realize it's not possible. Put the comment back in case we find out a way some day :smile: Custom protocol isn't implemented yet, so we can't really redirect back to Suite just now.